Dockerfile.centos

```diff
@@ -188,9 +192,15 @@ WORKDIR /ovms
 
 COPY src/ /ovms/src/
 COPY release_files/ /ovms/release_files/
+COPY set_version.sh /tmp
 
 ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/opt/intel/openvino/deployment_tools/inference_engine/lib/intel64/:/opt/intel/openvino/deployment_tools/ngraph/lib/:/opt/intel/openvino/inference_engine/external/tbb/lib/
 RUN ln -s /opt/intel/openvino_2021 /opt/intel/openvino
+RUN bash /tmp/set_version.sh REPLACE_PROJECT_NAME ${PROJECT_NAME}
+RUN if [ "$ov_use_binary" == "1" ] && [ -n $DLDT_PACKAGE_URL ] ; then true ; else exit 0 ; fi ;bash /tmp/set_version.sh REPLACE_OPENVINO_NAME `echo "$YUM_OV_PACKAGE" | sed -e 's/intel-openvino-runtime-centos7-//g' | sed 's/.x86_64//g'`
```

Comment:
you could use just a single sed like:
sed -r  's/.*-([0-9]+\.[0-9]+\.[0-9]+).x86_64/\1/'